Lib/doctest.py

```diff
@@ -1209,6 +1202,8 @@ def __init__(self, checker=None, verbose=None, optionflags=0):
         # Create a fake output target for capturing doctest output.
         self._fakeout = _SpoofOut()
 
+        self.debugger = None
```

Comment:
`self._debugger` might be better, but decided to keep it like this for some B/C at least.